src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/mercury/CuratedMainPagePageObject.java

```diff
@@ -38,15 +36,21 @@
 
     private int value;
 
-    private Settings(int value) {
+    Settings(int value) {
       this.value = value;
     }
   }
 
-  public CuratedPageObject(WebDriver driver) {
+  public CuratedMainPagePageObject(WebDriver driver) {
     super(driver);
   }
 
+  public int getElementHeight(String element) {
+    JavascriptExecutor js = (JavascriptExecutor) driver;
+    return Integer
+        .parseInt(js.executeScript("return $('" + element + "').offset().top").toString());
```

Comment:
what is more you can pass arguments to JS script. Thanks to this you can avoid string concatenation, instead just reference to argument by arguments[0] (for first argument) https://selenium.googlecode.com/git/docs/api/java/org/openqa/selenium/JavascriptExecutor.html